esp/services/ws_workunits/ws_workunitsHelpers.cpp

```diff
@@ -45,6 +45,12 @@
 
 namespace ws_workunits {
 
+#ifdef _CONTAINERIZED
+static const char* LOG_ACCESS_FEATURE = "WsLogAccess";
```

Comment:
this is fine, but the eclcc/thormaster logs are not related to wslogaccess at all. If we want to be sticklers we'd use WsLogAccess for component logs on containerized mode, and ClusterTopologyAccess for eclcc/thormaster/etc and baremetal component logs. Otherwise, we could use WsLogAccess for all logs and simplify things.

Reply:
On a call we followed the collection of logs for containerized and saw that the remote log reader is used for pulling all the logs (including eclcc and thormaster) in that case. So in the end it is reasonable to use these access flags.